brd-android/trade/src/main/java/com/fabriik/trade/ui/features/swap/AmountConverter.kt

```diff
@@ -40,10 +41,13 @@ class AmountConverter(
         val convertedAmount = sourceAmount.multiply(rate)
 
         val destFee = estimateFee(convertedAmount, destinationCurrency, fiatCurrency)
+
         val convertedDestAmount =
-            if (destFee?.included == true) convertedAmount - destFee.cryptoAmount.divide(receivingFeeRate) else convertedAmount
+            if (destFee?.included == false) {
+                Log.d("DAVID","included:" + true)
+                convertedAmount - destFee.cryptoAmount.divide(receivingFeeRate, 5, RoundingMode.HALF_UP)
```

Comment:
don't hardcode 5 decimals, some currencies use 6, some use up to 18, use the scale that the currency uses.

Reply:
I am not having that value for the currency I will use 20